src/specs/api.v2.yaml

```diff
@@ -1432,8 +1432,8 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/HTTPError"
-  ? "/experiments/{experimentId}/samples/{sampleId}/metadataTracks/{metadataTrackKey}"
```

Comment:
these ? signs here and earlier are weird. Is this even syntactically correct?

Reply:
no idea

Reply:
I think it was removed by the formatter